iOS-Play/iOS-Play/Controller/Tabar Controller/MainTabBarViewController.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  MainTabBarViewController.swift
+//  iOS-Play
+//
+//  Created by Quang Nguyen on 3/31/21.
+//
+
+import Foundation
+import UIKit
+
+class MainTabBarViewController: UITabBarController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        UITabBar.appearance().tintColor = .darkGray
```

Comment:
nên set cho instance của tabbar thôi, ko set như này
khi em set như này nó ăn global toàn app luôn